src/components/plots/abstractPlot.ts

```diff
@@ -208,6 +208,27 @@ export module Plot {
       return h;
     }
 
+    /**
+     * Generates a map from a projected attribute to the function to calculate the value of that attribute
+     * in accordance with the given datasetKey.
+     *
+     * Note that this will return all of the data attributes, which may not perfectly align to svg attributes
```

Comment:
Can we work with QE / Docs to come up with a clearer explanation of what this does?